components/infobox/wikis/starcraft2/infobox_league_hidden.lua

```diff
@@ -192,7 +192,8 @@ function HiddenInfoboxLeague._definePageVariables()
 
 	Variables.varDefine('tournament_game', (_GAMES[string.lower(_args.game or '')] or {})[1] or _GAMES[_GAME_WOL][1])
 
-	Variables.varDefine('tournament_parent', _args.parent)
+	-- Using current page title as parent is intended. See PR #1087 for background.
```

Comment:
It will be clear from git blame where this came from, so no need to mention a PR. I'd suggest this

```suggestion
	-- If no parent is available, set pagename instead to ease querying
```

Reply:
Updated as suggested.